introduction.md

```diff
@@ -1 +1,7 @@
 hey it's capital5, i like code school's javascript path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?